client-spark/common/src/test/java/org/apache/spark/shuffle/writer/WriteBufferManagerTest.java

```diff
@@ -41,14 +40,20 @@
 
 public class WriteBufferManagerTest {
 
+  static {
+    // trigger spark config package initialization before RssSparkConfig to avoid key, spark.shuffle.compress,
+    // duplication in spark3
+    org.apache.spark.internal.config.package$.MODULE$.EXECUTOR_MEMORY();
```

Comment:
Uniffle is shuffle plugin which gets startup after Spark internal config. As tested, there is no issue in real spark job. And the UT tests with spark2 and spark3 also passed without above code.

The only issue is you get duplicate key issue when you run the specific UT from GUI editor, like intellij. Thus, I added above code to avoid the issue.

Reply:
Let me just add string field to make thing easier.